.kokoro/build.sh

```diff
@@ -92,58 +104,51 @@ case ${JOB_TYPE} in
     fi
     ;;
   samples)
```

Comment:
You can delete this whole samples case. It's dead code.

Reply:
Yeah, the samples aren't testing anything right now. I thought we were fine with keeping this in preparation for the handwritten libraries moving into the monorepo. Are we going to come up with different way to test samples for handwritten libraries and I should just remove this block?

Reply:
Yeah, I think I said it's fine to keep, but seeing how complex the code is, I'd rather just get rid of it. We won't have to deal with handwritten samples in the future.

Reply:
Ok. I'll remove it.

Reply:
I've kept the samples.cfg for the kokoro job in for now. It just currently does nothing -- I'm considering deleting this config  and in g3 so it doesn't waste resources in our pool.